ddtrace/tracer/tracer.go

```diff
@@ -30,11 +30,15 @@ type tracer struct {
 	*config
 	*payload
 
-	flushAllReq    chan chan<- struct{}
-	flushTracesReq chan struct{}
-	exitReq        chan struct{}
+	// chanFlush triggers a flush of the buffered payload. If the sent channel is
+	// not nil (only in tests), it will receive confirmation of a finished flush.
+	chanFlush chan chan<- struct{}
```

Comment:
i think `flushChan` sounds more natural?

Reply:
It's true. The standard library also uses the `*Chan` suffix throughout. My intention was to group them, but since you emphasize this thought I already had, I'll go ahead and change all of them.